log4j-core/src/main/java/org/apache/logging/log4j/core/layout/AbstractStringLayout.java

```diff
@@ -158,17 +148,12 @@ protected static void trimToMaxSize(final StringBuilder stringBuilder) {
     /**
      * The charset for the formatted message.
      */
-    // LOG4J2-1099: Charset cannot be final due to serialization needs, so we serialize as Charset name instead
```

Comment:
This comment was no longer relevant given https://issues.apache.org/jira/browse/LOG4J2-1300 (5c6819530803357c8e580ad5143db4f2a4f33e64)